grakn-graph/src/main/java/ai/grakn/graph/internal/CastingImpl.java

```diff
@@ -65,9 +65,9 @@ public RoleType getRole() {
      * @return The {@link Instance} which is the roleplayer in this casting
      */
     public InstanceImpl getRolePlayer() {
-        Concept concept = getOutgoingNeighbour(Schema.EdgeLabel.ROLE_PLAYER);
-        if(concept != null) {
-            return (InstanceImpl) concept;
+        Optional<Concept> concept = getOutgoingNeighbours(Schema.EdgeLabel.ROLE_PLAYER).findFirst();
+        if(concept.isPresent()) {
+            return (InstanceImpl) concept.get();
         } else {
             return null;
```

Comment:
Ya I realised later I would use `return concept.orElse(null);` don't know why I did not go back and change this. Will do so now. Thanks!